src/openklant/components/contactgegevens/api/schema.py

```diff
@@ -9,4 +9,8 @@
     "TITLE": "contactgegevens",
     "DESCRIPTION": description,
     "VERSION": settings.CONTACTGEGEVENS_API_VERSION,
+    "TAGS": [
```

Comment:
The schemas probably have to be regenerated to add these tags?

Reply:
Yes. Updated OAS